pkg/ext-proc/scheduling/scheduler.go

```diff
@@ -110,13 +110,13 @@ type PodMetricsProvider interface {
 }
 
 // Schedule finds the target pod based on metrics and the requested lora adapter.
-func (s *Scheduler) Schedule(req *LLMRequest) (targetPod *backend.Pod, err error) {
+func (s *Scheduler) Schedule(req *LLMRequest) (targetPod backend.Pod, err error) {
```

Comment:
why did we change the return type here?

Reply:
The Pod object is a simple struct. In go, it's generally recommended to use the value directly for simple structs. This also makes logging the struct much easier.